pkg/workloads/caffe/caffe_test.go

```diff
@@ -31,7 +31,9 @@ func TestCaffeWithMockedExecutor(t *testing.T) {
 		mExecutor := new(executor.MockExecutor)
 		mHandle := new(executor.MockTaskHandle)
 
-		c := New(mExecutor, DefaultConfig())
+		config := DefaultConfig()
+		config.CollectAPM = false
```

Comment:
@skonefal - correct me if I'm wrong but it seems to me that we are missing test scenario for `CollectAPM` being `true`.